### PR TITLE
CC-32114 Added new CI files.

### PR DESCRIPTION
**Pull request by kkicheglovspryker**
## PR Description

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: .github/workflows/b2b-demo-marketplace-regression-ui-dev-robot.yml [hunk 1]
```diff
@@ -49,28 +42,29 @@ jobs:
...
+                  sudo bash -c "echo '127.0.0.1 backend-api.at.spryker.local backend-api.de.spryker.local glue-backend.de.spryker.local glue-backend.at.spryker.local glue-storefront.de.spryker.local glue-storefront.at.spryker.local backend-gateway.at.spryker.local backend-gateway.de.spryker.local backoffice.at.spryker.local backoffice.de.spryker.local date-time-configurator-example.spryker.local glue.at.spryker.local glue.de.spryker.local yves.at.spryker.local yves.de.spryker.local mp.de.spryker.local mail.spryker.local mp.at.spryker.local' >> /etc/hosts"
             - name: Custom commands
               continue-on-error: true
               run: |
                   docker/sdk cli composer dump-autoload -o -a
-                  APPLICATION_STORE=AT docker/sdk console queue:worker:start --stop-when-empty
+                  APPLICATION_STORE=DE docker/sdk testing console queue:worker:start --stop-when-empty
```
Comment (Nataliia-Popkova) [inline .github/workflows/b2b-demo-marketplace-regression-ui-dev-robot.yml:57]: are we sure that we need testing?
seams that we previously fox this

File: .github/workflows/b2c-demo-marketplace-regression-ui-dev-robot.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
-name: "[b2b-mp][internal] Robot Smoke UI"
+name: "[b2c-mp][internal] Robot Regression UI (DEV robot)"
 
 on:
     schedule:
-        - cron: '0 3 * * *'
+        -   cron: '0 3 * * *'
```
Comment (Nataliia-Popkova) [inline .github/workflows/b2c-demo-marketplace-regression-ui-dev-robot.yml:5]: we do not need schedule for DEV robot branch

File: .github/workflows/suite-regression-ui-dev-robot.yml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+            - name: Run Tests
+              run: |
+                  cd robotframework-tests
+                  rfbrowser init
+                  robot -v env:suite -v headless:true -v ignore_console:false -d results --exclude skip-due-to-issueORskip-due-to-refactoring tests/ui/e2e/suite.robot
+                  touch results/time.txt && echo $(date) > results/time.txt
+            - name: Upload artifacts
+              if: always()
+              run: |
+                  AWS_DEFAULT_REGION=${{env.ROBOT_TESTS_ARTIFACTS_BUCKET_REGION}} AWS_ACCESS_KEY_ID=${{ secrets.ROBOT_TESTS_ARTIFACTS_KEY }} AWS_SECRET_ACCESS_KEY=${{ secrets.ROBOT_TESTS_ARTIFACTS_SECRET }} aws s3 cp robotframework-tests/results s3://${{vars.ROBOT_TESTS_ARTIFACTS_BUCKET}}/robot-ui-testing/suite_regression/${GITHUB_RUN_ID}/PHP8.2MariaDBRobotUi/ \
```
Comment (Nataliia-Popkova) [inline .github/workflows/suite-regression-ui-dev-robot.yml:68]: incorrect folder for aws
should be /suite_regression_dev

**Review by Nataliia-Popkova — CHANGES_REQUESTED** [review]

File: .github/workflows/b2b-demo-marketplace-regression-ui-dev-robot.yml (hunk 1 above)
Comment (kkicheglovspryker) [reply .github/workflows/b2b-demo-marketplace-regression-ui-dev-robot.yml:57]: yes

**Pull request merged**